lib/entities/webhook.ts

```diff
@@ -182,6 +182,11 @@ export type WebhookProps = {
    * Transformation to apply
    */
   transformation?: WebhookTransformation
+
+  /**
+   * Whether the Webhook is active. If set to false, no calls will be made
+   */
+  active: boolean
```

Comment:
Make it optional?

Reply:
It's optional in the creation and update parameters, but always present in the response. 

See https://github.com/contentful/contentful-management.js/pull/1139/files#diff-03db97242908183de4e74d324bbb2877366c4cac40e405f9d65e6d247d404343R76-R80